src/plotman/manager.py

```diff
@@ -92,7 +106,7 @@ def maybe_start_new_plot(dir_cfg: plotman.configuration.Directories, sched_cfg:
                 for (d, phases) in eligible ]
 
         if not eligible:
-            wait_reason = 'no eligible tempdirs (%ds/%ds)' % (youngest_job_age, global_stagger)
+            wait_reason = 'waiting for phase match'
```

Comment:
Is this actually limited to indicating phase-related restrictions?  I quickly checked `phases_permit_new_job()` and it looks like it also checks the process limit.

https://github.com/ericaltendorf/plotman/blob/d52f51600736dbb2a1635ab80eede606949f1903/src/plotman/manager.py#L47-L74